releases.yml

```
@@ -1,3 +1,9 @@
+- version: 3.12.2
```

This version needs to be updated as well:
```suggestion
- version: 3.13.0
```

---

Ow! Well spotted. Fixed that as well.